app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_people_list = []
+    for person in people:
+        new_person = Person(person["name"], person["age"])
+        new_people_list.append(new_person)
+
+    for partner in new_people_list:
+        for person in people:
+            if partner.name == person["name"]:
+                if "wife" in person and person["wife"] is not None:
+                    partner.wife = Person.people[person["wife"]]
+                if "husband" in person and person["husband"] is not None:
+                    partner.husband = Person.people[person["husband"]]
```

Comment:
Why do you need 2 loops here. 
You can just iterate over `people` get current person by name from `Person.people` and set it's wife/husband in the same way you did it.